source/onboarding/elasticsearch/index.html.md.erb

```diff
@@ -0,0 +1,19 @@
+---
+title: ElasticSearch
```

Comment:
I don't think this is in the right location, didn't have time to review this fully before it was merged,

either way-of-working or even a new section, @njrich28 thoughts?